test/test_manager.py

```diff
@@ -946,6 +946,77 @@ def test_focus_stays_on_layout_switch(manager):
     assert manager.c.window.info()['name'] == 'one'
 
 
+@manager_config
+@no_xinerama
+def test_window_stacking_order(manager):
+    conn = xcbq.Connection(manager.display)
```

Comment:
You can use the `display` fixture instead, now.

Reply:
Sorry, I just borrowed the setup lines in here from other places in this file. I am having a somewhat hard time with the tests, the typeless-ness of Python makes understanding of some of these lines impossible outside of a debugger (and I have no setup at the moment to debug tests).

So your two comments on the test confuse me. Could you maybe elaborate a bit?